src/api/app/models/event/base.rb

```diff
@@ -287,7 +287,7 @@ def _roles(role, project, package = nil)
     end
 
     def send_to_bus
-      RabbitmqBus.publish(self.class.message_bus_routing_key, read_attribute(:payload))
+      RabbitmqBus.publish(self.class.message_bus_routing_key, self[:payload])
```

Comment:
Why is ``self[:payload]`` here necessary? Just ``payload`` should work as well, no?

Reply:
In general is better to use `self` if you want to be sure that you are setting the attribute and not a method that overrides what ActiveRecord defines.